File: C/D5/2/2.cpp
// Write a C program to for below output.
// A
// A B
// A B C
// A B C D

#include <stdio.h>

int main(){
    int i,j;

    for(i = 1; i <= 4; i++){
        for(int j = 1; j <= i; j++){
            printf("%c ", char(j+64));
        }
        printf("\n");      
    }
}

File: Python/D15 - T/2/2.py
# Write a python program to count uppercase characters and lowercase characters in string.

str = str(input('Enter a string : '))

space_count = str.count(' ')

str_no_space = str.replace(" ", "", space_count)

upper_count = 0
lower_count = 0

for i in str_no_space:
    upper_count += i.isupper()
    lower_count += i.islower()

print("upper_count = ", upper_count)
print("lower_count = ", lower_count)